### PR TITLE
Secret Management with Sealed Secrets

### PR DESCRIPTION
**Pull request by ernesgonzalez33**
Secrets used by default in Orion-LD and MongoDB (for the sake of the review, needs to be changed) and documentation on how to use the new chart

### CODE REVIEW

**Review by manuvaldi — APPROVED** [review]

File: documentation/GETTING_STARTED.md [hunk 1]
```diff
@@ -91,7 +91,11 @@ applications:
 > But this does not mean they are the right fit for a production ready deployment.
 > Please verify each application potential values (as all of them are Helm charts). You can either directly change the `values.yaml` of individual apps, or use the `values:` property directly in the app definition list in `fiware-platform/values.yaml` to override and/or set default values.
 
-### 3. Set the repo url in the values.yaml
+### 3. Decide if you want to use Sealed Secrets
+
+By default, all passwords needed for the deployment are in plain in the `values.yaml` file of each component. If you want to keep those secrets safe follow this [documentation](./SECRETS.md) and then continue from this point onward.
```
Comment (jtudelag) [inline documentation/GETTING_STARTED.md:96]: s/plain/plain text/ ?

File: documentation/SECRETS.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
+# Secrets
+
+> **NOTE:** For this moment, this feature is only compatible with **MongoDB** and **Orion-LD** charts.
+
+To manage the secrets in this repo we are using the Sealed Secrets approach by Bitnami. Sealed Screts allow you to store your password in a safe way event in a public repository due to they only can be decrypted by the controller running in the target cluster. If you want to know more, go to the Sealed Secrets Github repo at https://github.com/bitnami-labs/sealed-secrets.
```
Comment (jtudelag) [inline documentation/SECRETS.md:5]: Add a link to Sealed Secrets docs or repo.

File: secrets/values.yaml [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+  grafanaPassword: AgCUAkfP0YRSeM/OR43BkElJPj2L+JjVjaD20rITR610uulm5d+7wkoy7ds7uYARSrw5l6Pd99hYtQZof+Oj262TUPRu8hc7tkZnkHv6NS57l9XyGUkEYaat7m9Wm6WmUqSpV/5+yvhUfm3Jt7mxl73btVIK1vTo2ntj/MBvgKHXGwaczE13JS0ecl1+DK927sxru2nIedzcANmKimq5zJfudWdIE/mi8hs1/KlAVGz5w3jkBDSYAPlxNtX9ZeKt8cvWVKousuh+1QWLWRTJuPaK71nCnqY18lzUWOUTQKf/m1/3XbL4YLrE8uTf2c3AGRkGz5PWB5GA/34Mxtt38FU8tDWXWNhK01j6KtOv9s+Tgo/s4vV2Y+UBFxmVX3atd7arCCbxfakQUgSGA7nvpq3jvmUiz9PywgawTUkSC+0EDFkc8o1p80O82vrjBOugGNHYLtkF6qiDI7cPUs212heS+T5mDIJm5FqYUaKlRIqRx6K0ZHpK/W7rV6P5epgu6qSu3E6AqDEZpasjnETdrGH5dgd3HyILb3MJ3iLub7sS5jtGwPrqz31iWixAs22WZakF1p9gMaxuOFoMGoELAQYpZbxnECu5DymW+3AmunU9hpLmdRTgLcKFhi/4i45qg/0TsTf0CQIwMprNABI0OLGHWb+xEdVJEpYcDjIebv5SQpBWOUpqmdpAMtTnbqsDifO8PJhfxnH5+0hnQQ==
+
+  datasources:
+    timescale:
+      credentials:
+        password: *timescale-password
+
+  datasourcesNotEncrypted:
```
Comment (jtudelag) [inline secrets/values.yaml:19]: Cant we do sth about this? it is plain text here...

File: documentation/SECRETS.md (hunk 2 above)
Comment (ernesgonzalez33) [reply documentation/SECRETS.md:5]: It is there

File: secrets/values.yaml (hunk 3 above)
Comment (ernesgonzalez33) [reply secrets/values.yaml:19]: Where is the plain text? That's a link to the name of the password

**Comment by ernesgonzalez33** [comment]
Last check running means that tests e2e run with Mongo, Orion-LD, Grafana, and Grafana metrics using sealed secrets. The following commit will return to not using sealed secrets by default

File: documentation/GETTING_STARTED.md (hunk 1 above)
Comment (ernesgonzalez33) [reply documentation/GETTING_STARTED.md]: Done

**Pull request merged**